packages/runtime/client-api/src/api/document.ts

```diff
@@ -242,7 +242,7 @@ export async function load(
     urlResolver: IUrlResolver,
     options: ILoaderOptions = {},
     documentServiceFactory: IDocumentServiceFactory = defaultDocumentServiceFactory,
-    runtimeOptions: IContainerRuntimeOptions = { generateSummaries: false },
+    runtimeOptions: IContainerRuntimeOptions = {},
```

Comment:
Shouldn't `generateSummaries: false` be added as summary options here?

Reply:
Yep, good catch.